regulations/views/navigation.py

```diff
@@ -10,7 +10,10 @@ def _add_extra(el, version):
     """Add extra fields to a TOC element -- only added to elements we will
     use for prev/next"""
     if el.get('is_section'):
-        el['markup_prefix'] = '&sect;&nbsp;'
+        if el.get('is_section_span'):
+            el['markup_prefix'] = '&sect;&sect;&nbsp;'
+        else:
+            el['markup_prefix'] = '&sect;&nbsp;'
```

Comment:
Would you mind flattening this `if`? Maybe something like:
```python
if el.get('is_section_span'):
    ...
elif el.get('is_section'):
    ...
elif el.get('is_subterp'):
    ...
```

We should probably also switch these over to an enum rather than a bunch of bools but that's outside of the scope here.